src/deck/guides/konnect.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Using decK with Kong Konnect
+content_type: reference
+---
+
+You can manage {{site.base_gateway}} core entity configuration in your {{site.konnect_short_name}} organization using decK.
+
+Similar to Gateway workspaces, decK can only target one runtime group at a time.
+Managing multiple runtime groups requires a separate state file per group.
+
+You _cannot_ use decK to publish content to the Dev Portal, manage application registration, or configure custom plugins.
+
+## {{site.konnect_short_name}} flags
+
+You can use any regular `deck` commands (such as `ping`, `diff`, or `sync`) with `--konnect` flags to target {{site.konnect_short_name}} instead of a self-managed {{site.base_gateway}} control plane.
+
+You can also [save {{site.konnect_short_name}} credentials to a decK config file](#authenticate-with-a-deck-config-file) to target {{site.konnect_short_name}} by default.
+
+If you don't pass any {{site.konnect_short_name}} parameters to decK, decK looks for a local {{site.base_gateway}} instance instead.
+
+{:.note}
+> Prior to decK 1.12, decK provided `deck konnect` commands.
+These commands are deprecated and have been replaced with flags.
+
+`--konnect-addr`
+:  Address of the {{site.konnect_short_name}} endpoint. (Default: `"https://us.api.konghq.com"`)
+
+`--konnect-email`
+:  Email address associated with your {{site.konnect_short_name}} account.
+
+`--konnect-password`
+:  Password associated with your {{site.konnect_short_name}} account.
+This takes precedence over the `--konnect-password-file` flag.
+
+`--konnect-password-file`
+:  File containing the password to your {{site.konnect_short_name}} account.
+
+`--konnect-runtime-group-name`
+:  {{site.konnect_short_name}} runtime group name.
+
+## Authenticate with {{site.konnect_short_name}}
+
+decK looks for {{site.konnect_short_name}} credentials in the following order of precedence:
+
+1. Password set with the `--konnect-password` flag
+2. decK configuration file (default: `~/.deck.yaml`)
```

Comment:
The latter -  decK looks for a .deck.yaml file in the home directory, it doesn't create one. Do you think it would be useful to add that clarification?